### DISCUSSION TITLE
Core Registry Verbs

### DISCUSSION THREAD

**Discussion by jagadishbabu** [Sunbird-RC/community#36] [Sunbird RC Technical Specification Feedback]
@dileepbapat  
i seek clarification in terms of earlier documents referring to 7 core verbs - 

1. Enrol, 
2. Auth, 
3. Consent, 
4. Attest, 
5. Claim, 
6. Discover, 
7. Update.

Is the current sunbird RC complying with these 7 ( /less/more ).
Can there be one picture that can be added to the documentation that depicts this in a simple way.

The second request if - can the visual be provided to drive visual clarity along the lines mentioned below.

![image](https://user-images.githubusercontent.com/11401149/133192640-d6a5c0e4-8ed1-4519-80fb-7fa1aa634b50.png)

**Comment by dileepbapat** [comment]
Can I reorder Enrol, Authenticate, Update, Claim, Attest, Consent, Discover
